src/EA.Weee.Web/ViewModels/Shared/ViewEvidenceNoteViewModel.cs

```diff
@@ -48,5 +48,7 @@ public class ViewEvidenceNoteViewModel : EvidenceNoteViewModel
         public bool DisplayAatfName { get; set; }
 
         public bool DisplayH2Title { get; set; }
+
+        public int SelectedComplianceYear { get; set; }
```

Comment:
can you double check line 21 in this model, does         [DisplayName("Compliance year")]
        public string ComplianceYearDisplay => StartDate.Value.Year.ToString(); need updating?

Reply:
Yup will do another change to bring the compliance year from the db